package.json

```diff
@@ -27,13 +27,17 @@
     "@envelop/graphql-jit": "^1.1.1",
     "@envelop/parser-cache": "^2.2.0",
     "@envelop/validation-cache": "^2.2.0",
-    "@faststore/api": "^1.6.7",
-    "@faststore/sdk": "^1.6.7",
-    "@faststore/ui": "^1.6.7",
-    "@vtex/gatsby-plugin-nginx": "^1.6.7",
-    "@vtex/gatsby-plugin-thumbor": "^1.6.7",
-    "@vtex/gatsby-source-store": "^1.6.7",
-    "@vtex/graphql-utils": "^1.6.7",
+    "@faststore/api": "https://pkg.csb.dev/vtex/faststore/commit/56eafc23/@faststore/api/_pkg.tgz",
```

Comment:
just remember to update these packages before releasing. Also, don't add unecessary packages, like `gatsby-plugi-source-vtex`  etc

Reply:
Done!